drivers/scheduler/k8s/specs/postgres/px-postgres-app.yaml

```diff
@@ -37,12 +37,12 @@ spec:
           name: postgredb
         readinessProbe:
           exec:
-            command: ["psql", "-U", "pgbench", "-d", "pgbench", "-c", "SELECT 1"]
+            command: ["touch", "/var/lib/postgresql/data/pgdata/PG_VERSION"]
```

Comment:
@nikolaypopov can we try if we can do both?

i.e run a pgsql command first and then use `&&` and run the touch command.


This will ensure the the postgres DB is up and also ensure we can write to our mounted volume. Let's try this in your test cluster.

Reply:
Done.